### PR TITLE
mu8/santiago/issue4 Create domain module

### PR DESCRIPTION
**Pull request by SalosV**
Create domain module

### CODE REVIEW

**Comment by aliceresponde** [comment]
@SalosV  please change target branch in your PR to **development-mu8**

File: app/build.gradle [hunk 1]
```diff
@@ -24,8 +24,9 @@ android {
 }
 
 dependencies {
+    implementation project(":domain")
```
Comment (aliceresponde) [inline app/build.gradle:27]: swap lines 27 - 28

File: app/src/main/java/com/architect/coders/mu8/MainActivity.kt [hunk 2]
```diff
@@ -1,5 +1,4 @@
 package com.architect.coders.mu8
-
```
Comment (aliceresponde) [inline app/src/main/java/com/architect/coders/mu8/MainActivity.kt:2]: keep this blank line  (line2)

Comment (SalosV) [reply app/src/main/java/com/architect/coders/mu8/MainActivity.kt]: In the commit [13f7e](https://github.com/Architect-Coders/MU8/commit/13f7e12201dc7d22673362ddbf8f735c6ada8a1burl) it is already corrected.

**Pull request closed without merging**